packages/contracts/src/contracts/current/protocol/AssetTransferProxy/AssetTransferProxy.sol

```diff
@@ -0,0 +1,132 @@
+/*
+
+  Copyright 2018 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.4.21;
+
+import "./AssetProxyEncoderDecoder.sol";
+import "./IAssetProxy.sol";
+import "../../utils/Authorizable/Authorizable.sol";
+
+contract AssetTransferProxy is
+    AssetProxyEncoderDecoder,
+    Authorizable
+{
+    // Mapping from Asset Proxy Id's to their respective Asset Proxy
+    mapping (uint8 => IAssetProxy) public assetProxies_;
```

Comment:
Why not `assetProxies`?

Reply:
The trailing underscore denotes a state variable, although I see it's not used elsewhere in our contracts so I'll remove it.